### PR TITLE
Update ingress apiVersion to networking.k8s.io/v1

### PR DESCRIPTION
**Pull request by lgkgh**
Create GeoServer chart version v0.3.2 with updated ingress API version. More information can be found [here](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122).

### CODE REVIEW

File: charts/geoserver/v0.3.2/values.yaml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    # storageClass: "-"
+    accessModes:
+      - ReadWriteOnce
+    annotations: {}
+  geowebcacheCacheDir:
+    enabled: true
+    existingClaim: false
+    mountPath: /opt/geoserver/data_dir/gwc
+    subPath: "gwc"
+    size: 8Gi
```
Comment (NyakudyaA) [inline charts/geoserver/v0.3.2/values.yaml:103]: We are hardcoding the volume sizes i.e 8GI. Isn't there a way to define them to be generic i.e use env variable to enable a user to customise them ?

File: charts/geoserver/v0.3.2/values.yaml [hunk 2]
```diff
@@ -0,0 +1,132 @@
+image:
+  registry: docker.io
+  repository: kartoza/geoserver
+  tag: "2.17.0"
```
Comment (NyakudyaA) [inline charts/geoserver/v0.3.2/values.yaml:4]: I think we need to upgrade this to the latest version, Currently, GeoServer is at 2.21.0 but I haven't pushed the latest version to hub so maybe 2.20.4

File: charts/geoserver/v0.3.2/values.yaml (hunk 1 above)
Comment (lgkgh) [reply charts/geoserver/v0.3.2/values.yaml:103]: These are default values and can be updated before deployment.

File: charts/geoserver/v0.3.2/values.yaml (hunk 2 above)
Comment (lgkgh) [reply charts/geoserver/v0.3.2/values.yaml]: Tag updated.

**Comment by NyakudyaA** [comment]
LGTM @lgkgh

**Pull request merged**